build-gui/src/main/java/com/superzanti/serversync/GUI/Gui_JavaFX.java

```diff
@@ -30,7 +29,7 @@ public void start(Stage primaryStage) throws Exception {
         Scene scene = new Scene(root, 1024, 576);
         scene.getStylesheets().add(this.getClass().getResource("/css/application.bss").toExternalForm());
         primaryStage.setScene(scene);
-        primaryStage.setTitle(RefStrings.NAME + " - " + RefStrings.VERSION);
+       // primaryStage.setTitle(RefStrings.NAME + " - " + RefStrings.VERSION);
```

Comment:
Leftover debugging code? This is causing the window title to be missing and looks unrelated to build changes.

Reply:
Done here: https://github.com/superzanti/ServerSync/pull/319/commits/47fd704b46eb292326476f9f02d23e0f4c850b60